test/e2e/specs/editor/various/post-editor-template-mode.spec.js

```diff
@@ -378,4 +394,30 @@ class PostEditorTemplateMode {
 		);
 		await cancelButton.click();
 	}
+
+	/**
+	 * Edit the title of a template. The title is saved automatically, no
+	 * need to click anything, though we do click outside of the dialog
+	 * to close it.
+	 *
+	 * This needs to be called only after the template edit mode has been
+	 * activated or it will fail, i.e after creating a new custom template
+	 * or after calling `switchToTemplateMode()`.
+	 *
+	 * @param {string} newTitle the new title to set.
+	 * @return {Promise<void>}
+	 */
+	async editTemplateTitle( newTitle ) {
+		await this.page.click( 'button[aria-label="Template Options"]' );
```

Comment:
Perhaps this one can also be rewritten to use an accessible lotar `getByRole()` with `name: 'Template Options'`.

Reply:
Ah, thanks for the nudge about that!

Reply:
Done in https://github.com/WordPress/gutenberg/pull/50195/commits/c3808a3717133b6d71b03ea40d3e08a8340dcb77.